src/components/managementPanel/ManagementPanel.jsx

```diff
@@ -56,142 +88,207 @@ function ManagementPanel() {
     }
   }
 
-  const mapTiles = tiles.map(({ id, maxLat, maxLon, minLat, minLon, x, y }) => (
+  function getColor(id, usersCount) {
+    if (mouseOverTileData !== null && mouseOverTileData === id) {
+      return colors.colorMouseOverTile;
+    }
+
+    if (selectedTileData !== null && selectedTileData.id === id) {
+      return colors.colorTileActiveExplicit;
+    }
+    if (usersCount === 1) return colors.colorTileAssigned;
+    return colors.colorTileAll;
+  }
+
+  const mapTiles = tiles.map(({ id, maxLat, maxLon, minLat, minLon, x, y, usersCount }) => (
     <Rectangle
       key={`map-${id}`}
       bounds={[
-        [maxLat, maxLon],
-        [minLat, minLon],
+        [maxLat - 0.001, maxLon - 0.0015],
+        [minLat + 0.001, minLon + 0.0015],
       ]}
       pathOptions={{
-        color:
-          selectedTileData === null
-            ? colors.colorTileAll
-            : selectedTileData.id === id
-              ? colors.colorTileActiveExplicit
-              : colors.colorTileAll,
+        color: getColor(id, usersCount),
       }}
       eventHandlers={{
         click: async () => {
-          const response = await getTileUserAssignmentInfo(id);
-          if (response.status !== 200) {
-            setUserButtonTile('Unassigned');
-            setTileUsers([]);
-            return console.log('Api problem');
-          }
-          await setTileUsers(response.data.users);
-          setTileButtonTile(`X: ${x}, Y: ${y}`);
-          setSelectedTileData({ id, x, y });
+          handleTileSelected(id);
         },
+        mouseover: () => {
+          setMouseOverTileData(id);
+        }
       }}>
       <Tooltip direction="top">
-        x ={x}, y={y}
+        Y: {x}, Y: {y} <br />
+        {t('managementPanel.assigned')}: {usersCount === 1 ? t('yes') : t('no')} <br />
       </Tooltip>
     </Rectangle>
   ));
 
-  const tilesList =
+  const handleTileSelected = async (tileId) => {
+    loadingUsersState();
+    const selectedTile = tiles.find(x => x.id === tileId);
```

Comment:
You can use destructuring to keep code neater

Reply:
Yea, I'll try.